### DISCUSSION TITLE
How do you manage data from various jurisdictions in a statewide WZDx data feed?

### DISCUSSION THREAD

**Discussion by hua-xiang** [usdot-jpo-ode/jpo-wzdx#102] [Q&A]
Hi, the wonderful WZDx community: 

Do you have experience of managing data from various jurisdictions in a statewide WZDx data feed? Or, maybe I should ask how you collaborated with other state/local jurisdictions when you set up the WZDx feed. Your lessons learned will help us start the process in the right way.

Thank you!
Hua

**Comment by lynnerandolph** [comment]
For TxDOT, the various districts are using C2C to pull data into a centralized location. That information contains data from 25 districts mostly using the TxDOT Lonestar ATMS software for putting in lane closure information. One district uses a different software, but with C2C, their data is converted into the C2C format and combined with the other districts. We also pull in data from a third party who is handling work zones on I-35 in the Waco area, again converting to C2C format. Since we already had those disparate sources integrated together using C2C, we then just created a conversion to the WZDx format and published that as a separate feed.

**Reply by hua-xiang** [reply]
Thank you, Lynne for, the answer! Does a feed user see a difference when you publish it as a separate feed after conversion?

**Reply by semosher** [reply]
Each one of TxDOT's districts has a unique id. The district id of each road event is contained in the "subidentifier" element. For example, a road event originating in Austin would have subidentifier "AUS".

**Comment by fsaleemmaricopa** [comment]
In Maricopa County, AZ we leveraged the AZTech regional collaboration for integrating the local agency work zone data into our central Regional Archived Data Platform. We ingest data in the format provided by the agency and curate it centrally. Attached is the high level representation.

[AZTech WZ Data Sharing.pptx](https://github.com/usdot-jpo-ode/jpo-wzdx/files/4801105/AZTech.WZ.Data.Sharing.pptx)

**Reply by hua-xiang** [reply]
Thank you, Faisal. Does the Maricopa County maintain the central regional archived data platform? What's the biggest hurdle that you have to overcome when you curate the local agency data? 

Also, an unrelated question: does AZ511 offer voice assistance features based off Google, Amazon Alexa, or Apple Siri? Just curious.

**Comment by fsaleemmaricopa** [comment]
You're welcome!

Yes we maintain a central data platform that serves as central archive as well regional data processing and dissemination mechanism.

Initially we approached the local agencies to provide data in regional/national standard format but it required double entry as agencies have to enter once on their own agency platform and also on regional platform. Some agencies do not  have resources to support the regional platform. We changed our approach and created processes for pulling the data in automated way from their respective platforms. The challenge with centrally curating is common nomenclature as agencies may report the same road in different ways e.g. Loop 101, L101, L-101 etc. We worked with agency partners to address this issue and provided guidance. Other issue is some agencies do not provide GIS data and we had to create processes centrally to add it.

ADOT recently moved to a new 511 system. I will have to check on the voice features.


Thank you,

Did I provide excellent service? Tell us how we are doing.<http://apps.mcdot.maricopa.gov/customersurvey/?SRID=2>

Faisal Saleem
ITS Branch Manager
O: 602.506.1241 ▪ C: 602.723.6766
Maricopa County Department of Transportation<http://mcdot.maricopa.gov/>
2901 W. Durango Street ▪ Phoenix, AZ 85009
Faisal.Saleem@Maricopa.Gov<mailto:traciruth@mail.maricopa.gov>



________________________________
